tests/test.js

```diff
@@ -0,0 +1,141 @@
+const junit = require('junit');
```

Comment:
Rename the file to e.g. `splitting.test.js`

Reply:
Done